biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -31,38 +36,237 @@ br {
   width: 56.25rem;
 }
 
+#intro {
+  background: #adadad;
+  border: 5px solid black;
+  border-radius: 5px;
+  color: rgb(0, 0, 0);
+  padding: 10px;
+}
+
 label {
   display: block;
   text-align: left;
 }
 
 .largeBatteryCells {
   display: inline-block;
+  margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
 }
 
 .smallBatteryCells {
   display: inline-block;
-  margin-right: 1.25rem;
   margin-bottom: 1.25rem;
+  margin-left: 1.25rem;
+  margin-right: 1.25rem;
+  margin-top: 1.25rem;
```

Comment:
`margin: 1.25rem;`

Reply:
looks like this is applied in a lot of places. What about renaming this to a more generic class like "grid item" and having all grid items have this class instead of one class per item?

Reply:
Thanks will fix that.